{{cookiecutter.repo_name}}/tox.ini

```diff
@@ -65,10 +65,10 @@ commands =
     {%- endif %}
 {%- else %}
     {%- if cookiecutter.test_matrix_separate_coverage|lower == 'yes' %}
-    nocov: {posargs:nosetests -v tests}
```

Comment:
Hmmm... on second thought, does the `[nosetests]where` in `setup.cfg` always apply?